### DISCUSSION TITLE
API Issue: page update needs to specify "path" and "tag" as required parameters in the document

### DISCUSSION THREAD

**Discussion by michelleyuan-qualcomm** [requarks/wiki#3634] [Error / Bug Report]
**Describe the bug**
For the wiki.js PageMutation update() function, besides the “id” parameter, we also need to specify "path" and “tags” parameters. If the path is not provided, we get "you are not authorized to update this page". If the tags are not specified, we get “Cannot read property 'map' of undefined”. 

**To Reproduce**
If the path is not provided in the parameters: I get the following error.
{
  "data": {
    "pages": {
      "update": {
        "responseResult": {
          "succeeded": false,
          "message": "You are not authorized to update this page."
        }
      }
    }
  }
}

If "tags" are not specified, I get the following error:

{
  "data": {
    "pages": {
      "update": {
        "responseResult": {
          "succeeded": false,
          "message": "Cannot read property 'map' of undefined"
        }
      }
    }
  }
}

**Expected behavior**
If both "path" and "tags" are required parameters for PageMutation update() function, we can update the schema in the document to reflect that. 

**Screenshots**

**Host Info (please complete the following information):**
OS: Ubuntu 18.04
Wiki.js version: 2.5.144
Database engine: PostgresQL

**Additional context**

**Comment by iautom8things** [comment]
@michelleyuan-qualcomm Thank you for reporting this issue nearly _exactly_ a year ago.  It's unreal that such a simple fix wasn't implemented then, and still hasn't been.

**Comment by tombayo** [comment]
This other discussion about the same issue seems to be the most active:
https://github.com/requarks/wiki/discussions/3627

Perhaps this discussion can be marked as answered, and any further discussion may continue in the one linked above.
